package.json

```diff
@@ -20,12 +20,12 @@
     "db:studio": "prisma studio",
     "setup": "run-s db:generate db:push db:seed",
     "start": "cross-env NODE_ENV=production node ./build/server.js",
-    "start:mocks": "cross-env NODE_ENV=production node --require ./mocks --require dotenv/config ./build/server.js",
+    "start:mocks": "cross-env NODE_ENV=production node --require dotenv/config ./build/server.js",
```

Comment:
What is the expected behavior of this? Currently I get `Unexpected Server Error` at `localhost:8080`, with these logs:

```
TypeError: (0 , import_jsx_dev_runtime.jsxDEV) is not a function
    at /home/shawn/starchart/app/entry.server.tsx:27:9
    at new Promise (<anonymous>)
    at handleRequest (/home/shawn/starchart/app/entry.server.tsx:21:10)
    at handleDocumentRequestRR (/home/shawn/starchart/build/server.js:39728:24)
    at requestHandler (/home/shawn/starchart/build/server.js:39591:22)
    at /home/shawn/starchart/build/server.js:40748:26
GET / 500 - - 65.736 ms
```

Reply:
We don't even use these mocks, should we get rid of this?

Reply:
I don't get this error:
<img width="471" alt="image" src="https://user-images.githubusercontent.com/67077705/216664425-765f49b1-68f2-4140-aa60-159b494196c6.png">

Could you please list the exact steps you did when you got this?

Reply:
1. Check out your PR
2. run `npm i`
3. run `npm run start:mocks`
4. navigate to `locahost:8080`. See `Unexpected Server Error`
As well as see this in the terminal:
![image](https://user-images.githubusercontent.com/55592708/216676381-e35f8243-6bc4-4c93-be32-4dc5a92f87ea.png)